packages/components/src/toolbar/toolbar/index.tsx

```diff
@@ -40,6 +40,9 @@ function UnforwardedToolbar(
 			Dropdown: {
 				variant: 'toolbar',
 			},
+			Menu: {
+				variant: 'toolbar',
+			},
```

Comment:
This change is necessary since, with the rename, the `Menu` component "connects" to the components context system under the `Menu` namespace (previously it was `DrodpdownMenu`)

Reply:
Could be a storybook-related issue, i'll look into it.

Reply:
Yup, I confirm that the Storybook example on `trunk` is currently broken (regression introduced in #64654) — only the Storybook example is broken, not the component's integration with `Toolbar`.

The example is back to work as expected in this PR.